Event monitor uses event explorer for all sites (#13795)
DataDog/documentation: content/en/monitors/create/types/event.md

  text: "Check your monitor status"
---

## Overview

Event monitors allow you to alert on events matching a search query.

## Monitor creation

> To create an [event monitor][1] in Datadog, use the main navigation: *Monitors --> New Monitor --> Event*.

Reviewer: ```suggestion
To create an [event monitor][1] in Datadog, navigate to **Monitors** > **New Monitor** > **Event**.
```

### Define the search query

As you define the search query, the top graph updates.

1. Construct a search query using the [Event Explorer search syntax][2].
2. Choose to monitor over an event count, facet, or measure:
    * **Monitor over an event count**: Use the search bar (optional) and do **not** select a facet or measure. Datadog evaluates the number of events over a selected time frame, then compares it to the threshold conditions.
    * **Monitor over a facet**: If a facet is selected, the monitor alerts over the unique value count of the facet.
    * **Monitor over measure**: If a measure is selected, the monitor alerts over the numerical value of the event facet (similar to a metric monitor) and aggregation needs to be selected (`min`, `avg`, `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`).
3. Configure the alert grouping strategy (optional):
    * **Simple-Alert**: Simple alerts aggregate over all reporting sources. You receive one alert when the aggregated value meets the set conditions. This works best to monitor a metric from a single host or the sum of a metric across many hosts. This strategy may be selected to reduce notification noise.

>     * **Multi-Alert**: Multi alerts apply the alert to each source according to your group parameters, up to 100 matching groups. An alerting event is generated for each group that meets the set conditions. For example, you could group by `host` to receive a separate alert for each host.

Reviewer: ```suggestion
    * **Multi-Alert**: Multi alerts apply the alert to each source according to your group parameters, up to 100 matching groups. An alerting event is generated for each group that meets the set conditions. For example, you can group by `host` to receive separate alerts for each host.
```
